src/components/atoms/Modal.tsx

```diff
@@ -1,20 +1,28 @@
 import { css } from '@emotion/react';
 
 interface ModalProps {
+  padding?: string;
+  closable: boolean;
```

Comment:
요거 닫기 버튼 자체를 받는 건 어떠신가유? 그러면 더 유연한 구조가 될 것 같습니다!

```suggestion
  closeButton?: ReactNode;
```

Reply:
977efaa 에 반영했습니다!